File: Data Structures & Algorithms/concatenation-of-array/submission-0.java
class Solution {
    public int[] getConcatenation(int[] nums) {
        int n= nums.length;
        int[] arr= new int[2*n];
        int j= n;
        for(int i= 0; i<n; i++){
            arr[i]= nums[i];
            arr[j]= nums[i];
            j++;
        }
        return arr;
    }
}

File: Data Structures & Algorithms/majority-element-ii/submission-0.cpp
class Solution {
public:
    vector<int> majorityElement(vector<int>& nums) {
        vector<int> ans;
        unordered_map<int, int> freq;
        for(int num: nums){
            freq[num]++;

        }
        int n= nums.size();
        for(auto x: freq){
            if(x.second>n/3){
                ans.push_back(x.first);
            }
        }
        return ans;
    }
};